powdr_cli/src/main.rs

```diff
@@ -54,6 +56,11 @@ enum Commands {
         #[arg(short, long)]
         #[arg(default_value_t = false)]
         force: bool,
+
+        /// Generate a halo2 proof
```

Comment:
I think it would be nice to already start with an enum instead of just a bool

Reply:
With the different backends?

Reply:
Changed to `Option<Backend>`